src/db/migrations/1550932229367-CreateChallenges.ts

```diff
@@ -0,0 +1,22 @@
+import { MigrationInterface, QueryRunner, Table } from "typeorm";
+
+export class CreateChallenges1550931581297 implements MigrationInterface {
```

Comment:
Both filename and class name were generated by TypeORM CLI. I think we don't need to change this.

Reply:
I asked because the CreateWipChallenges1551228127302 class has the same timestamp as its file name. If TypeORM does not require the class name to match exactly the file name, there's really no reason to change anything.

Reply:
Makes sense. TypeORM does not require but I agree. Maybe was some copy and paste. Changed :+1: